lib/markdown/markdown.nit

```diff
@@ -856,9 +864,31 @@ class HTMLDecorator
 	private var allowed_id_chars: Array[Char] = ['-', '_', ':', '.']
 end
 
+# Location in a Markdown input.
+class MDLocation
+
```

Comment:
I think that you should also add the filepath or something (if any) because:

* you do want to help the user to manage all these metadata, and the original source file is.
* It may help you when you will deal with import/include/inline of markdown in markdown to track the original file within the location